packages/terra-clinical-result/package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "terra-clinical-result",
+  "main": "lib/ClincalResult.js",
+  "version": "0.0.0",
+  "description": "A set of components for displaying Clinical Observations like Vital Signs and Laboratory Results.",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/cerner/terra-clinical.git"
+  },
+  "keywords": [
+    "Cerner",
+    "Terra",
+    "Clinical",
+    "terra-clinical-result",
+    "ClinicalResult",
+    "Result",
+    "Observation",
+    "UI"
+  ],
+  "author": "Cerner Corporation",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/cerner/terra-clinical/issues"
+  },
+  "homepage": "https://github.com/cerner/terra-clinical#readme",
+  "peerDependencies": {
+    "react": "^16.8.5",
+    "react-dom": "^16.8.5"
+  },
+  "dependencies": {
+    "classnames": "^2.2.5",
+    "prop-types": "^15.5.8",
+    "terra-doc-template": "^2.12.0",
+    "terra-icon": "^3.0.0",
+    "terra-mixins": "^1.0.0"
+  },
+  "scripts": {
+    "compile": "babel --root-mode upward src --out-dir lib --copy-files",
+    "lint": "npm run lint:js && npm run lint:scss",
+    "lint:js": "eslint --ext .js,.jsx . --ignore-path ../../.eslintignore",
+    "lint:scss": "stylelint src/**/*.scss",
+    "precompile": "rm -rf lib",
+    "test": "npm run jest && npm run wdio",
+    "jest": "jest --config ../../jestconfig.js",
+    "wdio": "cd ../.. && npx wdio ./packages/$npm_package_name/wdio.conf.js"
```

Comment:
There should be wdio scripts for each theme: https://github.com/cerner/terra-clinical/blob/master/packages/terra-clinical-item-view/package.json#L45-L48

Reply:
Added https://github.com/cerner/terra-clinical/pull/574/commits/9eeacc2469da67d5f2d9ad3ce7cc533a407dbf13?file-filters%5B%5D=.js&file-filters%5B%5D=.json&file-filters%5B%5D=.jsx&file-filters%5B%5D=.scss&file-filters%5B%5D=.snap#diff-f98243208c330dd1438f9a62456b5a8fR45